src/main/java/application/dto/order/OrderRequestStatusDto.java

```diff
@@ -0,0 +1,9 @@
+package application.dto.order;
+
+import application.model.Order;
+import lombok.Data;
+
+@Data
+public class OrderRequestStatusDto {
+    private Order.Status status;

```

Comment:
Just wondering if you use this enum in several places outside of the `Order` class, probably that would be better to move it to the separate file?